client/utils.c

```diff
@@ -759,3 +759,41 @@ TDNFGetCmdOpt(
     return dwError;
 }
 
+uint32_t
+TDNFReadAutoInstalled(
+    PTDNF pTdnf,
+    char ***pppszAutoInstalled
+    )
+{
+    uint32_t dwError = 0;
+    char **ppszAutoInstalled = NULL;
+    char *pszAutoFile = NULL;
+
+    if (!pTdnf || !pppszAutoInstalled)
+    {
+        dwError = ERROR_TDNF_INVALID_PARAMETER;
+        BAIL_ON_TDNF_ERROR(dwError);
+    }
+
+    dwError = TDNFAllocateStringPrintf(&pszAutoFile, "%s/%s/%s",
+            pTdnf->pArgs->pszInstallRoot,
```

Comment:
check if install root is "/", string formation might end up in "//.."

Reply:
Thx - that's what `TDNFJoinPath()` is for.